gdal/doc/source/development/rfc/rfc79_listing_service_providers.rst

```diff
@@ -0,0 +1,124 @@
+
+In this RFC, we propose a plan to address this by adding a Service Providers
+page to the GDAL website.
+
+2. Proposed plan
+================
+
+A new Service Providers page will be added to the Sphinx documentation.
+
+The service providers will be grouped in three categories. The descriptions
+below should make it relatively easy to determine in which group a given
+organization belongs.
+
+Each qualified organization who wishes to be listed is responsible for
+adding themselves to the right category in the page through a pull
+request (or a direct commit in case of core committers). We are intentionally
+not going to provide instructions here for preparing the pull request as
+it is expected that a qualified service provider should have people on
+staff who can figure it out themselves. Yes, we intentionally
+try to keep the bar at a minimum level.
+
+A PSC vote is NOT required in order to add a new service provider,
+i.e. any core committer can validate and commit the pull
+request (or their own entry) but in case of doubts or disagreements on the
+interpretation of the rules, the GDAL PSC will be the ultimate authority
+to resolve the question.
+
+Organizations in each category are displayed in randomized order (except
+in the PDF docs which are static). However entries should be added to the
+source file in alphabetical order (for PDF docs purpose).
+The output will be similar to
+https://mapserver.org/community/service_providers.html
+
+2.1 Core Contributors
+---------------------
+
+Core Contributors are essentially the organizations that have one or more
+Core Committer (a person with merge rights in the GDAL GitHub repository)
+and/or PSC members as part of their staff and who provide
+GDAL related services to customers.
+
+To qualify, the Core Committers and/or PSC members must be directly employed
+by the organization and not simply part time contractors.
+
+They get to add a 200px wide logo with a descriptive text outlining their qualifications and services.
+
+2.2 Contributors
+----------------
+
+Contributors are organizations who provide GDAL related services and
+have one or more people on their staff who are well known in the GDAL
+community for their long term contributions to the project. Contributions are
+not limited to source code and can also be in the form of documentation,
+binary builds, active user support via the public forums, etc.
+
+They get to add a 150px wide logo with a descriptive text outlining their qualifications and services.
+
+2.3 Other Service Providers
+---------------------------
+
+Organization with GDAL expertise and providing services around GDAL
+can fit in this category. While proprietary solutions providers are welcome,
+we also expect organizations listed here to be providing legitimate services to
+users of the Open Source software as well and don't want this
+to just become an advertisement board for proprietary products.
```

Comment:
"well and don't want this to just become an advertisement" -> "well. This is not  an advertisement"